client/src/utils/swr-hooks.ts

```diff
@@ -49,10 +49,6 @@ export const fetcherGET: Fetcher<any, string> = (url) =>
     },
   }).then((res) => {
     if (!res.ok) {
-      if (res.status === 401) {
```

Comment:
Fjerner sideeffekt i en metode som skal hente noe fra server. Hvis en error blir kastet så bør vi heller sette en globalError der den brukes hvor 401 blir håndtert :)

Reply:
Så vi skal bruke globalError og håndtere 401 overalt vi gjør ett kall til server? :) Husker ikke hva vi ble enige om her bare

Reply:
GlobalError er satt opp til å håndtere 401.. Vi må alltid håndtere responsen fra server. Hvis responsen er 401, 403 eller 500 kan vi sette en globalError siden dette er feil som vi alltid vil håndtere likt?. Noen steder vil vi kanskje håndtere andre feil lokalt i komponenten (som skjemafeil f.eks.), andre ganger kanskje nok å sette en globalError for alle statuskoder. GlobalError viser en modal som kanskje er litt rart i noen tilfeller. Vi må tenke oss om ved alle kall i hvert fall 😇 Hvis det var svar på spm? 😅